client/packages/common/src/ui/layout/tables/material-react-table/useMaterialTableColumns.tsx

```diff
@@ -52,10 +52,10 @@ export const useMaterialTableColumns = <T extends MRT_RowData>(
         }
 
         return {
+          grow: true,
           ...columnDefaults,
           enableSorting: col.enableSorting ?? false,
-          enableColumnFilter:
-            col.enableColumnFilter ?? !!col.filterVariant ?? false, // if a filter variant was explicitly set, take that as shorthand to enable the column filtering
```

Comment:
originally has enabling filtering if you define `filterVariant` on a column, but I think its easier to reason about a set of column definitions, if you can just scan through, see all the ones with `enableColumnFilter` on, and know those are the filterable columns